__tests__/__packages__/cli-test-utils/package.json

```diff
@@ -46,6 +46,6 @@
     "@zowe/imperative": "8.0.0-next.202311132045"
   },
   "peerDependencies": {
-    "@zowe/imperative": "^5.0.0"
+    "@zowe/imperative": "^8.0.0"
```

Comment:
According to the [npm semver calculator](https://semver.npmjs.com/), `^8.0.0` doesn't match versions with a prerelease string like `8.0.0-next.XXX`.

I think `^8.0.0-next` would work for now. In the future once after we've successfully published a `next` tag, we may want to switch to `>=8.0.0-next.XXX <9.0.0` and reference a specific timestamp - I believe this is what we settled on doing back when v2 was vNext.